docs/index.md

```diff
@@ -2,23 +2,22 @@
+* Delegation, where a delegate's privileges derive from a chain of authorizations that must extend back to a valid root authority.
+* Statistical inference and legal judgements, where multiple sources of data may assert contradictory facts, and the ability to weigh cumulative evidence is vital.
+* Citation of sources (in art, in journalism, in academia, or in credential issuance), where an author wants to clarify that a particular assertion originates elsewhere. This allows the assertion to acquire (or lose) gravitas independent of the reputation of the author who repeats it, and it lets reusers of data be held accountable for compliance with data sharing constraints.
 
-The last of these examples deserves special comment. There is a tension between the decentralization that we want in a verifiable credential ecosystem, and the way that trust tends to centralize because knowledge and reputation are unevenly distributed. We want anyone to be able to attest to anything they like--but we know that verifiers care very much about the reputation of the parties that make those attestations.
+The last of these examples deserves special comment. There is a tension between the decentralization that we want in a verifiable data ecosystem, and the way that trust tends to centralize because knowledge and reputation are unevenly distributed. We want anyone to be able to attest to anything they like &mdash; but we know that verifiers care deeply about the reputation of the parties that make those attestations.
 
-We could say that verifiers will choose which issuers they trust. This is exactly what most practioners of VCs recommend, and it works in early pilots. However, this places a heavy burden on verifiers, over the long haul--verifiers can't afford to vet every potential issuer of credentials they might encounter. The result will be a tendency to accept credentials only from a short list of issuers, which leads back to centralization.
+We could say that verifiers will choose which issuers they trust. This is exactly what most practitioners of verifiable credentials recommend, and it works fine in early pilots. However, as the complexity of a data ecosystem grows, it doesn't scale; verifiers simply can't afford to vet every potential issuer of credentials they might encounter. (Consider a timely example. Tens of thousands of labs are now issuing COVID test results. No verifier can possibly know them all.) One obvious remedy is to accept credentials only from a short list of issuers, which leads back to centralization. Another remedy is to create a trust registry, where vetted issuers can be looked up. This introduces centralization in a different way.
 
```

Comment:
"This introduces centralization in a different way" 

I would suggest to talk more about federated approach or as we call it fractal governance where you decide how this federation can be build up and bridge between governance authority. This could give a feeling that on highest level you have sort of centralization (to bridge) but it does not impact local governance at all. So would not call it centralization in that case.